.circleci/config.yml

```diff
@@ -66,23 +66,15 @@ jobs:
           name: Run codegen
           command: yarn codegen
 
-      # For all Node targets, run everything -- except tests/bun and tests/esm-misc (Jest doesn't load `jest.config.ts` in Node 18/20)
+      # For all Node targets, run everything -- except tests/bun (run in a separate job)
```

Comment:
Node 20 can run ESM tests, so I'm pulling this back into the main test suite runtime.